src/main/java/com/ruleoftech/markdown/page/generator/plugin/MdPageGeneratorMojo.java

```diff
@@ -281,6 +301,31 @@ private int getPegdownExtensions(String extensions) {
         return options;
     }
 
+    private <T> MutableDataHolder getFlexmarkParserOptions(String options) {
+        MutableDataHolder optionsAsDataKeys = new MutableDataSet();
+
+    	if (options == null)
+    		return optionsAsDataKeys;
+    	
+        for (String opt : Arrays.asList(options.split("\\s*,\\s*"))) {
+            try {
+                Field f = Parser.class.getField(opt);
+                @SuppressWarnings("unchecked")
+				DataKey<T> dataKey = (DataKey<T>)f.get(null);
+                optionsAsDataKeys.set(dataKey, dataKey.getDefaultValue(null));
+                getLog().info("Flexmark option " + opt);
```

Comment:
These log statement, being at info level, has made me hesitate to upgrade to the 2.3.0 release, as the build server logs start to fill up with the same status information each time I call the plugin.